project-templates/template-README.md

```diff
@@ -0,0 +1,11 @@
+Project Foo
+===========
+
+Foo is a library to do something useful.
```

Comment:
shouldn't we say something along the line "Foo is a registered [Vulkan Ecosystem](link to the ecosystem) project"?